README.md

```diff
@@ -223,6 +223,14 @@ the `year_of_ius_passing_Xpct_under_threshold` is the year all simulated IUs rea
 - **processed_prevalence** - the composite prevalence in Africa. The prevalence for a specific draw is worked out by
   taking the prevalence for each IU we have results for, multiplying it by its priority population, summing across the
   IUs, then dividing by the priority population of all included IUs.
+- **prob_all_ius_under_threshold** - probability of disease elimination. It is worked out
+  by 
+  - identifying the draws below the prevalence threshold across all IUs
+  - checking if all IUs, for a given year-draw combination, are below threshold
```

Comment:
💚 It took me two read throughs to get it, but honestly I think two isn't bad, and I certainly can't think of a more natural way to express it!